### PR TITLE
GetKeyIndex is nop since 1.87+ and obsoleted from 1.90.5

### PR DESCRIPTION
**Pull request by sonoro1234**
tittle explain all the main issue
Also had to add `#include <cmath>` to three files in order to avoid `sqrt not defined`

### CODE REVIEW

File: ImGuiDebugPanel.cpp [hunk 1]
```diff
@@ -1,3 +1,4 @@
+#include <cmath>
 #include "TextEditor.h"
```
Comment (santaclose) [inline ImGuiDebugPanel.cpp:2]: where are we using sqrt in this file? i have the same question regarding `LanguageDefinitions.cpp` and `UnitTests.cpp`

Comment (sonoro1234) [reply ImGuiDebugPanel.cpp:2]: It is used inside TextEditor.h

Comment (santaclose) [reply ImGuiDebugPanel.cpp:2]: why not add the include there then?

Comment (sonoro1234) [reply ImGuiDebugPanel.cpp:2]: Yes.

Comment (sonoro1234) [reply ImGuiDebugPanel.cpp]: Done

**Comment by sonoro1234** [comment]
I dont know if I can combine in one commit and push to have this PR with only one commit?

**Comment by santaclose** [comment]
Hi! yes i would appreciate if you can squash the commits into one, i think you can do `git reset HEAD~4` to soft reset to the revision before your changes, and then create a new commit combining the changes. Then you can do `git push <remote> <branch> --force` to update your fork with the squashed commit

**Comment by sonoro1234** [comment]
Not so good on git!!! ( I use TortoiseGit for the work ...)
But i will make one way or the other ;-)

**Comment by sonoro1234** [comment]
Just made another PS

**Pull request closed without merging**